utils/librarypath/librarypath.go

```diff
@@ -39,6 +40,21 @@ func cleanPath(paths []string, fullPath bool) []string {
 	return processedPaths
 }
 
+func RemoveIgnoredPath(paths []string, ignoredPaths []string) []string {
+	var result []string
+	result = utils.Filter(paths, func(s string) bool {
+		var result bool = false
+		for _, value := range ignoredPaths {
+			result = strings.Contains(strings.ToUpper(s), strings.ToUpper(value))
```

Comment:
I think using strings.HasPrefix is better for now, based on the relative path of the project folder.